system/mp/mpcomm.cpp

```diff
@@ -756,8 +760,7 @@ protected: friend class CMPPacketReader;
                 // loop with short wait time and release CS to allow other side to proceed
                 StringBuffer epStr;
                 unsigned startMs = msTick();
-
-                unsigned loopCnt = ((CONNECT_READ_TIMEOUT / retrycount) / CONNECT_TIMEOUT_INTERVAL) + 1;
+                unsigned loopCnt = CONNECT_TIMEOUT_LOOPCNT;
```

Comment:
>loopCnt is now 20x larger than before, right ?

Yes... but only at the start...
Your code made the overall time a lot small than it originally was..

It made:
1st retry pass = 10 secs (loopCnt = 10 * 1sec timeouts)
2nd retry pass = 10 secs (rounded down)
3rd retry pass = 11 secs
..
10th loop pass = 18 secs
...
19th loop pass = 180 secs

As I say, it climbed as retryCount fell.

The original pre-HPCC-12128 code, would take [up to] 180 secs per pass.